File: spec/code_samples/Javascript_jquery/list@cleanse@{list_id}/get.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://analyzemail.com/api/v1/list/cleanse/%7Blist_id%7D?username=SOME_STRING_VALUE",
  "method": "GET",
  "headers": {
    "content-type": "application/json"
  }
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: spec/code_samples/Python_python3/list@cleanse@{list_id}/get.py
import http.client

conn = http.client.HTTPSConnection("analyzemail.com")

headers = { 'content-type': "application/json" }

conn.request("GET", "/api/v1/list/cleanse/%7Blist_id%7D?username=SOME_STRING_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))